internal/export/distro/registrations.go

```diff
@@ -328,7 +331,18 @@ func Loop(errChan chan error, eventCh chan *models.Event) {
 				LoggingClient.Warn(fmt.Sprintf("Error updating registration %s", update.Name))
 			}
 
-		case event := <-eventCh:
+		case msgEnvelope := <-messages:
+			LoggingClient.Info(fmt.Sprintf("Event received on message queue. Topic: %s, Correlation-id: %s ", Configuration.MessageQueue.Topic, msgEnvelope.CorrelationID))
+			if msgEnvelope.ContentType != clients.ContentTypeJSON {
+				LoggingClient.Error(fmt.Sprintf("Incorrect content type for event message. Received: %s, Expected: %s", msgEnvelope.ContentType, clients.ContentTypeJSON))
+				continue
+			}
+			str := string(msgEnvelope.Payload)
+			event := parseEvent(str)
+			if event == nil {
```

Comment:
We don't want to handle an error in any way other than skipping it?

Reply:
parseEvent() logs the error if it can't parse